readme.md

```diff
@@ -91,7 +91,7 @@ For reference, the following run time dependencies are included in Git submodule
 * [eSpeak NG](https://github.com/espeak-ng/espeak-ng), version 1.52-dev commit `f520fecb`
 * [Sonic](https://github.com/waywardgeek/sonic), commit 1d705135
 * [IAccessible2](https://wiki.linuxfoundation.org/accessibility/iaccessible2/start), commit cbc1f29631780
-* [liblouis](http://www.liblouis.org/), version 3.24.0
+* [liblouis](http://www.liblouis.org/), version 3.25.0
```

Comment:
According to release notes, the website's URL has changed:

```suggestion
* [liblouis](http://www.liblouis.io/), version 3.25.0
```

Please also look in NVDA's codebase if there are other occurrences of this link. At least there is one in `copying.txt`.

It's worth noting that the old .org website is indicated as not secure by my browser.